src/lib/zod/index.ts

```diff
@@ -39,6 +48,10 @@ export const circleIdInput = z
   })
   .strip();
 
+export const uploadImageInput = z
+  .object({ image_data_base64: z.string() })
+  .strict();
```

Comment:
The name of this variable makes it sound like it's a function that will upload an image. Might be better to Capitalize it instead and name it `UploadImageInput` or `UploadImageInputSchema`